agimus_controller/agimus_controller/warm_start_base.py

```diff
@@ -13,7 +13,7 @@ def __init__(self) -> None:
     def generate(
         self,
         reference_trajectory: list[TrajectoryPoint],
-    ) -> tuple(np.ndarray, np.ndarray):
+    ) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
There is also a `import numpy.typing as npt`? Or something like that.
@Kotochleb gave a lot of feedback on this previous PR.

Reply:
I found comments from  @Kotochleb, is there a way to accept them somehow? Cause I see there were commit suggestions, but they were never commited

Reply:
They were NOT accepted? Well. It doesn't stop me from creating 50 new coments